File: Practice/Beginner/Solved/FCTRL.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int T;	scanf("%d",&T);
	while(T--)
	{
		int n;	scanf("%d",&n);
		long long int p=5,ans=0;;
		int m=1;
		while(1)
		{
			int fac=n/p;
			if(fac==0)
				break;
			ans+=(fac);
			p=p*5;
		}	
		printf("%lld\n",ans);
	}
}

File: Practice/Beginner/Solved/FCTRL.py
T=int(input())
while T:
	n=int(input())
	p=5
	ans=0
	m=1
	var=1
	while var==1:
		fac=int(n/p)
		if fac==0:
			var=0
		ans+=fac
		p*=5
	print(ans)
	T=T-1